packages/forms-web-app/src/controllers/appellant-submission/submission.js

```diff
@@ -35,15 +31,6 @@ exports.postSubmission = async (req, res) => {
 		return;
 	}
 
-	const isWithinExpiryPeriod =
-		decisionDate &&
-		validation.appeal.decisionDate.isWithinDecisionDateExpiryPeriod(new Date(decisionDate));
-
-	if (!isWithinExpiryPeriod) {
-		res.redirect(`/${VIEW.ELIGIBILITY.DECISION_DATE_PASSED}`);
-		return;
```

Comment:
we do not need this logic - we check for date expiry on the middleware

also we do not perform a check like this in the v1 full appeal submission controller (and the specified redirect page here is legacy and now removed).